### PR TITLE
refactor: Fair coverage for docker exec utils

### PR DESCRIPTION
**Pull request by zharinov**
## Changes:

Add `index.spec.ts` for `util/docker`

## Context:

Ref: #10526

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/util/exec/docker/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+import {
+  generateDockerCommand,
+  getDockerTag,
+  prefetchDockerImage,
+  removeDanglingContainers,
+  removeDockerContainer,
+  resetPrefetchedImages,
+} from '.';
+
+const rawExec: jest.Mock<typeof _rawExec> = _rawExec as any;
```
Comment (viceice) [inline lib/util/exec/docker/index.spec.ts:16]: Why not using `test/exec-util` as in other tests?

**Review by viceice — COMMENTED** [review]
Otherwise LGTM

File: lib/util/exec/docker/index.spec.ts (hunk 1 above)
Comment (zharinov) [reply lib/util/exec/docker/index.spec.ts:16]: Haha, I forgot we have it 😅

File: lib/util/exec/docker/index.spec.ts [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+  exec,
+  mockExecAll,
+  mockExecSequence,
+} from '../../../../test/exec-util';
+import { getName } from '../../../../test/util';
+import { setAdminConfig } from '../../../config/admin';
+import { SYSTEM_INSUFFICIENT_MEMORY } from '../../../constants/error-messages';
+import { getPkgReleases as _getPkgReleases } from '../../../datasource';
+import { logger } from '../../../logger';
+import { VolumeOption } from '../common';
```
Comment (viceice) [inline lib/util/exec/docker/index.spec.ts:11]:
```suggestion
import type { VolumeOption } from '../common';
```

**Review by viceice — COMMENTED** [review]
just a small neat

**Review by viceice — APPROVED** [review]

**Pull request merged**